python/cudf/dataframe/dataframe.py

```diff
@@ -421,26 +421,18 @@ def __repr__(self):
             len(self),
         )
 
-    # binary, rbinary, unary, orderedcompare, unorderedcompare
-    def _call_op(self, other, internal_fn, fn):
+    # binary, rbinary, orderedcompare, unorderedcompare
+    def _apply_op(self, fn, other):
```

Comment:
Since I don't have the power to request a review, I'm tagging @thomcom and @kkraus14

Reply:
👍